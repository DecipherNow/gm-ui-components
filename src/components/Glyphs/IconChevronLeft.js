import React from "react";
import Icon from "../Icon";

export default function ChevronLeft(props) {
  return (
    <Icon {...props} glyphName="ChevronLeft">
      <path
        d="M8.4696699,7.53033007 C8.17677668,7.23743685 8.17677668,6.76256311 8.4696699,6.46966989 C8.76256312,6.17677667 9.23743686,6.17677667 9.53033008,6.46966989 L15.0606602,12 L9.53033008,17.5303301 C9.23743686,17.8232233 8.76256312,17.8232233 8.4696699,17.5303301 C8.17677668,17.2374369 8.17677668,16.7625632 8.4696699,16.4696699 L12.9393398,12 L8.4696699,7.53033007 Z"
        id="ChevronLeft"
         
        fillRule="nonzero"
        transform="translate(11.655330, 12.000000) scale(-1, 1) translate(-11.655330, -12.000000) "
      />
    </Icon>
  );
}

ChevronLeft.displayName = "ChevronLeft";
