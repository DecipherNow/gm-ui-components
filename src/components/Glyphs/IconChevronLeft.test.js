import React from "react";
import { shallow } from "enzyme";

import IconChevronLeft from "./IconChevronLeft";

describe("IconChevronLeft", () => {
  it("matches snapshot", () => {
    const aIconChevronLeft = shallow(<IconChevronLeft />);
    expect(aIconChevronLeft).toMatchSnapshot();
  });
});