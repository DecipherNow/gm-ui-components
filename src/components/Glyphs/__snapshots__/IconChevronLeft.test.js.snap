// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconChevronLeft matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="ChevronLeft"
  glyphSizeRatio={1}
>
  <path
    d="M8.4696699,7.53033007 C8.17677668,7.23743685 8.17677668,6.76256311 8.4696699,6.46966989 C8.76256312,6.17677667 9.23743686,6.17677667 9.53033008,6.46966989 L15.0606602,12 L9.53033008,17.5303301 C9.23743686,17.8232233 8.76256312,17.8232233 8.4696699,17.5303301 C8.17677668,17.2374369 8.17677668,16.7625632 8.4696699,16.4696699 L12.9393398,12 L8.4696699,7.53033007 Z"
    fillRule="nonzero"
    id="ChevronLeft"
    transform="translate(11.655330, 12.000000) scale(-1, 1) translate(-11.655330, -12.000000) "
  />
</Icon>
`;
